data/beps/beps.yml

```
@@ -597,3 +597,38 @@
     google_doc_created: 2023-09-12
     pull_request_created:
     pull_request_merged:
+
+
+-   number: '046'
+    title: Cardiac MRI (CMR)
+    google_doc: [[https://docs.google.com/document/d/1StWtkrQpJyBjPcPjAMrk07QJcoG2gGzih8LX0lhmGKI/edit]]
+    leads:
+    -   given-names: Edward
+        family-names: Xu
+        # mailto:zexu@ottawaheart.ca
+    -   given-names: Jefferson
+        family-names: Casimir
+        # @jeffersoncasimir
+    -   given-names: Ghislain
+        family-names: Vaillant
+        # @ghisvail
+    -   given-names: Christine
+        family-names: Rogers
+        # @christinerogers
+    bids_maintainers:
+    -   given-names: Christine
+        family-names: Rogers
+        # @christinerogers
```

```suggestion
    -   Name: [Edward Xu](mailto:zexu@ottawaheart.ca)
         # @Edward-ZX
    -   Name: Jefferson Casimir
        # @jeffersoncasimir
    -   Name: Ghislain Vaillant
        # @ghisvail
    -   Name: Christine Rogers
        # @christinerogers
    bids_maintainers:
    -   Name: Christine Rogers
        # @christinerogers
```
updating names, adding email, to better match [other BEP examples](https://github.com/bids-standard/bids-website/pull/287/files)